rust/ql/test/library-tests/type-inference/loop/main.rs

```diff
@@ -9,6 +9,6 @@ trait T1<T>: T2<S<T>> {
 
 trait T2<T>: T1<S<T>> {
     fn bar(self) {
-        self.foo()
+        self.foo() // $ method=foo
```

Comment:
I think we should change this to using canonical paths, once that PR lands, because the comment as-is doesn't really tell us anything about the target.

Reply:
Yes, we could do that if they're not too cumbersome to write.

But this also works reasonably well. We only use just the name in the cases where there's a single method of a given name in scope, and then the method resolution would have to do something very wrong to find another method with the same name.